docs/4.1/enterprise/ssh_fips.md

```diff
@@ -0,0 +1,150 @@
+  # Pre-defined tokens for adding new nodes to a cluster. Each token specifies
+  # the role a new node will be allowed to assume. The more secure way to
+  # add nodes is to use `ttl node add --ttl` command to generate auto-expiring
+  # tokens.
+  #
+  # We recommend to use tools like `pwgen` to generate sufficiently random
+  # tokens of 32+ byte length.
+  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
+  auth_servers: [ "10.1.1.10:3025" ]
+
+auth_service:
+  # enable the auth service:
+  enabled: true
+
+  tokens:
+  # this static token is used for other nodes to join this Teleport cluster
+  - proxy,node:zw6C82kq7VEUSJeSDzuldWsxakql6jrTYmphxRQOlrATTGbLQoaIwEBo48o9
+  # this token is used to establish trust with other Teleport clusters
+  - trusted_cluster:TaZff3DLbpsMZmIMhvEr7kulOgegjg7yyQNTS0q6UFWfsJ9N6rxVBjg6t7nw
+
+  # To Support FIPS local_auth needs to be turned off and a SSO connector is 
+  # required to log into Teleport. 
+  authentication:
+    # local_auth needs to be set to False in FIPS mode. 
+    local_auth: false
+    type: saml
+
+  # If using Proxy Mode, Teleport requires host key checks. 
+  proxy_checks_host_keys: true
+
+  # SSH is also enabled on this node:
+ssh_service:
+  enabled: false
+```
+
+### Teleport Node
+
+Save the following configuration file as `/etc/teleport.yaml` on the _node.example.com_:
+```yaml
+teleport:
+  auth_token: zw6C82kq7VEUSJeSDzuldWsxakql6jrTYmphxRQOlrATTGbLQoaIwEBo48o9
+
+  auth_servers: [ "10.1.1.10:3025" ]
+
+  # enable ssh service and disable auth and proxy:
+ssh_service:
+  enabled: true
+
+auth_service:
+  enabled: false
+proxy_service:
+  enabled: false
+```
+
+### Systemd Unit File
+
+Next, download the systemd service unit file from [examples directory](https://github.com/gravitational/teleport/tree/master/examples/systemd/fips)
+on Github and save it as `/etc/systemd/system/teleport.service` on both servers. 
+
+```bsh
+# run this on both servers:
+$ sudo systemctl daemon-reload
+$ sudo systemctl enable teleport
+```
+
+### Starting Teleport in FIPS mode.
+
+Using `teleport start --fips` Teleport will start in FIPS mode, Teleport will 
+configure the TLS and SSH servers with FIPS compliant cryptographic algorithms. 
+In FIPS mode, if non-compliant algorithms are chosen, Teleport will fail to start.
+In addition, Teleport checks if the binary was compiled against an approved
+cryptographic module (BoringCrypto) and fails to start if it was not.
+
+* For OSS and Enterprise binaries not compiled with BoringCrypto, this flag will report that this version of Teleport is not compiled with the appropriate cryptographic module.
+
+* Be useful when running commands like ps aux to note that Teleport is running in FedRAMP enforcing mode.
+
+* If no ciphersuites are provided, set the default ciphersuites to be FIPS 140-2 compliant. 
+
+* If the ciphersuites are provided in Teleport configuration, validate they are FIPS 140-2 compliant.
```

Comment:
"If ciphersuites are provided in the Teleport configuration, Teleport will validate that they are FIPS 140-2 compliant."